payjoin-test-utils/src/lib.rs

```diff
@@ -36,21 +38,26 @@ pub fn init_tracing() {
 
 pub struct TestServices {
     cert_key: (Vec<u8>, Vec<u8>),
+    #[allow(dead_code)]
+    redis: (u16, Container<'static, Redis>),
```

Comment:
Perhaps add a docstring comment `/// redis is an implicit dependency of directory [on the default port]` to justify the #[allow(dead_code)] or otherwise set `PJ_DB_HOST` explicitly from redis's port info